1 Python/docs/Classes and OOP/py-classes-basic.md

```diff
@@ -0,0 +1,67 @@
+# Basic Classes and Objects
```

Comment:
I'm not sure about this entire directory of additional Python OOP info.  I've always told the students it was here if they wanted to search for something, but never really looked into it myself.  I don't know if it's valuable or just makes the repo messier.

Reply:
That is a folder of OOP docs another instructor wrote ~3 years ago. I've always kept it in as extra info but I never actually teach out of it. I think it's good to have as many OOP resources for students, but I don't know if anytone actually ever goes in there.

Reply:
I agree, If they want to find additional information we could point them to the official python docs instead.